src/front-end/src/components/MentorForm/MentorForm.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { mockWindowMatchMedia } from '../../helper-methods';
+
+import MentorForm from './MentorForm';
+
+let container = null;
+
+mockWindowMatchMedia();
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+test('render without crashing', () => {
+  render(<MentorForm />, container);
+  expect(container.textContent).toBe(
+    'Sign up as a MentorOrganizationNameEmailTime-zoneSelect timezone..PasswordConfirm PasswordTop 3 skills\xa0DescriptionSubmit',
+  );
+});
+
+test('Every text field has its input field except', () => {
```

Comment:
Sorry, I don't understand this test name, can you make it clearer? Is there a word missing after 'except'?